Source/Core/Common/VR/DolphinVR.cpp

```diff
@@ -0,0 +1,239 @@
+#include <cstring>
+
+#include "Common/VR/DolphinVR.h"
+#include "Common/VR/VRBase.h"
+#include "Common/VR/VRInput.h"
+#include "Common/VR/VRRenderer.h"
+
+/*
+================================================================================
+
+VR controller mapping
+
+================================================================================
+*/
+
+enum ControlID
```

Comment:
Please don't redefine an existing enum. This will lead to hard-to-catch problems if the numbering is ever changed in one of the enums but not the other.